app/src/main/res/values/styles.xml

```diff
@@ -37,6 +37,23 @@
 
     </style>
 
+    <style name="courseTheme" parent="Theme.MaterialComponents.Light.DarkActionBar">
+        <!-- Customize your theme here. -->
+        <item name="colorPrimary">@color/colorPrimary</item>
+        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
+        <item name="colorAccent">@color/colorAccent</item>
+        <item name="android:fontFamily">@font/nunitosans_semibold</item>
+        <item name="android:windowTranslucentStatus">true</item>
+        <item name="windowActionBarOverlay">true</item>
+        <item name="android:textColor">@color/black</item>
+        <item name="android:statusBarColor">@color/white_transparency</item>
+        <item name="android:navigationBarColor">@color/colorPrimaryDark</item>
+        <item name="windowActionBar">false</item>
+        <item name="windowNoTitle">true</item>
+        <item name="android:windowFullscreen">true</item>
```

Comment:
why full screen? the navigation bar also extra theme params please remove which are not needed